lib/koans/18_genservers.ex

```diff
@@ -12,9 +12,13 @@ defmodule GenServers do
       {:ok, args}
     end
 
-    def start_link(init_password) do
+    def start(init_password) do
       # The __MODULE__ macro returns the current module name as an atom
-      GenServer.start_link(__MODULE__, init_password, name: __MODULE__)
+      GenServer.start(__MODULE__, init_password, name: __MODULE__)
+    end
+
+    def stop do
+      GenServer.stop(__MODULE__)
```

Comment:
✍️ I've opted to manually manage the process instead of relying on linking. Since each koan is run in its own process, there was a race condition between the servers when running linked. Since koans are tested in order, this should clean up after itself, but even if it doesn't I think we're pretty safe as overall runtime is short.